vsphere/virtual_machine_config_structure.go

```diff
@@ -262,6 +262,12 @@ func schemaVirtualMachineConfigSpec() map[string]*schema.Schema {
 			Computed:    true,
 			Description: "The UUID of the virtual machine. Also exposed as the ID of the resource.",
 		},
+		"latency_sensitivity": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Default:     "normal",
+			Description: "The latency-sensitivity setting of the virtual machine.",
```

Comment:
Maybe include the possible values here?

http://pubs.vmware.com/vsphere-6-5/index.jsp#com.vmware.wssdk.apiref.doc/vim.LatencySensitivity.SensitivityLevel.html